rsmtool/reader.py

```diff
@@ -260,7 +260,7 @@ def read_from_file(filename, converters=None, **kwargs):
 
         # ignore warnings about mixed data types for large files
         with warnings.catch_warnings():
-            warnings.filterwarnings('ignore', category=pd.io.common.DtypeWarning)
```

Comment:
Would this change affect using earlier versions of `pandas`?

Reply:
Good catch! Yes, it will break for older versions of pandas. So we should pin pandas to be >=1.0.5 where this change was introduced?

Reply:
Yeah, that would probably be the easiest way to handle this. But, if we want to allow for earlier versions of `pandas`, perhaps we could have a try/except at the top to import `DtypeWarning` from first `pandas.errors` and then, if that fails, `pandas.io.common` and then we could just use `DtypeWarning` as the category here.

Reply:
I like that idea much better!